Networking/Networking/Settings/UserAgent.swift

```diff
@@ -1,36 +1,28 @@
-import Foundation
-import UIKit
 import WebKit
 
-
 /// WooCommerce User Agent!
 ///
-class UserAgent {
-
-    /// Private: NO-OP
-    ///
-    private init() { }
-
+public class UserAgent {
 
-    /// Returns the WooCommerce User Agent
+    /// Returns the default WooCommerce iOS User Agent
     ///
-    static var defaultUserAgent: String = {
+    public static var defaultUserAgent: String = {
         return webkitUserAgent + " " + Constants.woocommerceIdentifier + "/" + bundleShortVersion
     }()
 
     /// Returns the WebKit User Agent
     ///
-    static var webkitUserAgent: String {
+    public static var webkitUserAgent: String {
```

Comment:
nit: this property could probably be internal to Networking? it seems to be used only in unit tests

Reply:
👍 fixed here f94baa4368d619cff3379dbc13b2d1cbf1f288e3